.gitignore

```diff
@@ -137,10 +137,16 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
-# Mine
+# Tony's py
 
+<<<<<<< HEAD
```

Comment:
This is an unresolved merge conflict.

You MUST review your prs after you submit them.